tasks/Intergroup Prisoner's Dilemma-Maximizing Difference.md

```diff
@@ -0,0 +1,64 @@
+
+# Procedure
+## Steps
+Intergroup Prisoner’s Dilemma(IPD)
+- Six participants are split into two groups, with three people per group, no communication between participants
+- Each participant is taken to their own room, and given instructions to the game
+- Participants are given a short quiz that tests their knowledge of the game
+    - If communication was allowed, participants were allowed to meet with their group members for 5 minutes of free discussion, then return to their rooms to make their decisions, otherwise skip this
+- They are then asked to distribute their tokens amongst the options, their choice is then filled out on a decision sheet
+- Participants are then asked to fill out a post game questionnaire assessing their expectations
+
+Intergroup Prisoner’s Dilemma - Maximizing Difference(IPD-MD)
+- Six participants are split into two groups, with three people per group, no communication between participants
+- Each participant is taken to their own room, and given instructions that use neutral language (pools are referred to as A and B)
+- Participants are given a short quiz that tests their knowledge of the game
+    - If communication was allowed, participants were allowed to meet with their group members for 5 minutes of free discussion, then return to their rooms to make their decisions, otherwise skip this
+- They are then asked to distribute their tokens amongst the options, their choice is then filled out on a decision sheet
+- Participants are then asked to fill out a post game questionnaire assessing their expectations
+
+## Roles 
+Participant, is placed in a group at random and plays the game as instructed
+
+## Instructions
+Intergroup Prisoner’s Dilemma(IPD)
+- Six participants are split into two groups, with three people per group, no communication between participants
+- Each participant gets 10 tokens, and can contribute as many as they want to the group
+    - For each token contributed to the group, everyone in the group gains 1 money unit(MU), and every member of the out group loses 1 MU
+    - For each token kept by a participant, the participant is paid 2 MU
+
+Intergroup Prisoner’s Dilemma - Maximizing Difference(IPD-MD)
+- Six participants are split into two groups, with three people per group, no communication between participants
+- Each participant gets 10 tokens, each token is worth 2 MU, and can contribute as many as they want to one of the two pools, the “within-group pool”(pool W) or the “between-group pool”(pool B)
+    - For each token contributed to pool W, everyone in the group gains 1 MU
+    - For each token contributed to poo B, everyone in the group gains 1 MU, and everyone in the other group loses 1 MU
+
+- Alternate version allows for communication within the group before making their independent decisions.
+    - If communication was allowed, participants were allowed to meet with their group members for 5 minutes of free discussion, then return to their rooms to make their decisions
+
```

Comment:
Participants were explicitly assured that their decisions would remain confidential. I think this is important to add since it can affect the way decisions are made.